### PR TITLE
For approver sign and unsign, it will now update the row and not the …

### PR DESCRIPTION
**Pull request by ValdarRudman**
…entire table. The wait has been removed as well

### CODE REVIEW

**Comment by ValdarRudman** [comment]
Is there a another way to disable the button instead of a tag?
I tried using event, but it gives me a pointer event and I was unable to find the location to disable.
Used to be event.target.disabled or something like that

**Comment by ValdarRudman** [comment]
Not sure if opacity is the best. Does not show well on the white background.

If you have some time tomorrow evening, I can show you.

**Comment by SeanCondon** [comment]
> Not sure if opacity is the best. Does not show well on the white background.
> 
> If you have some time tomorrow evening, I can show you.

If you even have a screenshot send it on

File: src/app/users-signed/users-signed.component.css [hunk 1]
```diff
@@ -73,6 +73,11 @@ mat-footer-row mat-footer-cell {
     overflow: hidden;
     vertical-align: middle;
 }
+<<<<<<< Updated upstream
```
Comment (SeanCondon) [inline src/app/users-signed/users-signed.component.css:76]: merge issue?

**Review by SeanCondon — COMMENTED** [review]
I think you have some merge issues here in the `css` - I thought `ng lint` should have caught this

**Comment by ValdarRudman** [comment]
Was going to upload screenshots today, but my node modules folder broke. Reinstalled but app wont run.

I'll upload screenshot when this is resolved

File: src/app/users-signed/users-signed.component.css (hunk 1 above)
Comment (ValdarRudman) [reply src/app/users-signed/users-signed.component.css:76]: this happened as i kept committing to the wrong branch.
I caught the first time. Sorted now

**Comment by ValdarRudman** [comment]
Normal
image

With opacity 0.5
image

With a grey background and opacity 0.5
image


The grey background looks better.

Maybe try a row outline of some kind.
To use opacity, colour will need to be added

**Comment by SeanCondon** [comment]
thanks for screen shots - I think the 0.5 opacity on its own looks fine - if there were others that were your own reportees next to it, it would be distinctive. mayve 0.6 or 0.7 would be better

**Comment by ValdarRudman** [comment]
Kept 0.5 opacity. See how it goes.

I change to flex. It keeps the scrolling of the table as well as the sticky header. Graphical issues were happening when using the non flex way.

This also keeps the mobile view useable

**Review by SeanCondon — APPROVED** [review]
LGTM

**Pull request merged**